Code-250/my-brand
# qsCTLuxi-admin-page

### What does this PR do?

This pull request implements the CRUD operations for blog and skills

### Description of Task to be completed?

- able to see querries and questions
- able to navigate to create an article
- able to navigate to list of all articles
- able to see delete and edit article buttons
- able to create and edit an article
- able to delete article
- able to update an article

### How should this be manually tested?

- `git clone to have the repository to your local machine
- `git checkout ft-admin-page-qsCTLuxi` to go to the branch that you can use to test this.
- run `admin.html ` in your browser
- Click on the article link on the sidebar
- click on create blog to see a model
- Click on add button to add a skill.

### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

- [article-page](https://trello.com/c/qsCTLuxi/15-admin-dashboard)

### Screenshots (if appropriate)
- desktop list article view
![desktop](https://user-images.githubusercontent.com/47631065/147367420-b0aa8826-5163-4217-a7f2-efefa3fdc463.png)


- contact messages view
![image](https://user-images.githubusercontent.com/47631065/147367586-0f9d117b-03e5-4162-a64a-cd113396cb4b.png)


### Questions:
N/A
